### ISSUE TITLE
UXD - Clarify regions that are not part of LA city on Map

### ISSUE THREAD

**Issue by nichhk** [hackforla/311-data#1305]
### Overview
We need to clarify when a user is hovering over an area that is NOT a neighborhood council because users are getting confused about what they are selecting, leading to a bad and confusing experience with our app.

### Releases
- [ ] https://github.com/hackforla/311-data/issues/1585

### Action Items

- [x] Address the problem overview with possible design solution(s) in a comment
- [x] Complete Design Iterations section below
- [x] Document user interaction in Figma
- [x] Update Hand Off Materials below

---

### Design Iterations

**Please move ticket between `In Progress` and `In Review` to assist PM team**

<details><summary>Iteration 1</summary>
<p>

Link to notes: https://github.com/hackforla/311-data/issues/1305#issuecomment-2725834708

<img width="773" alt="Image" src="https://github.com/user-attachments/assets/3ea02000-4299-45cb-a88e-b414a0f0f2c9" />

<img width="392" alt="Image" src="https://github.com/user-attachments/assets/8d0c43b0-3f23-46a3-8ff2-706293ee6ad5" />

<img width="900" alt="Image" src="https://github.com/user-attachments/assets/249b1e92-f99f-4cc9-ba50-d45be75a1ed2" />

</p>
</details> 

<details><summary>Iteration 2</summary>
<p>

Link to notes: https://github.com/hackforla/311-data/issues/1305#issuecomment-2725846150

<img width="1173" alt="Image" src="https://github.com/user-attachments/assets/c392e09d-c78c-49e7-8252-ba1fb92a5b83" />

</p>
</details> 

<details><summary>Iteration 3</summary>
<p>

Link to notes: https://github.com/hackforla/311-data/issues/1305#issuecomment-2744693472
- Added darker opaque border to non-NC boundaries to be more consistent with NC-area boundary designs
- Included example for 'Greater Los Angeles' area popup that for areas that are not detailed in https://github.com/hackforla/311-data/issues/1305#issuecomment-1459202092
<img width="1437" alt="Image" src="https://github.com/user-attachments/assets/5e4cdbad-941c-474e-849e-92b2a4f40198" />

</p>
</details> 
---

### Hand Off Materials

Figma Section Name: `⭐⭐⭐ Handoff`

<details><summary>Before Screenshot</summary>
<p>

<img width="1545" alt="Image" src="https://github.com/user-attachments/assets/1a7f07aa-c755-478e-8ef4-c23e7ca7deb1" />

</p>
</details> 

<details><summary>After Screenshot (Finalized)</summary>
<p>

<img width="1151" alt="Image" src="https://github.com/user-attachments/assets/12b63509-5bb2-4573-99ed-69ce89af2072" />

</p>
</details> 

### Designer Resources

<details><summary>Iteration Dropdown Copy/Paste</summary>
<p>

```
<details><summary>Iteration X</summary>
<p>

Link to notes: `REPLACE WITH COMMENT URL`

`REPLACE WITH SCREENSHOT UPLOAD`

</p>
</details> 
```

</p>
</details> 

<details><summary>Instructions for Engineering Hand Off</summary>
<p>

To Start Engineering Hand Off...
1. Ensure all Hand Off Materials are filled in
3. Add the "ready for dev lead" label
4. Leave a comment saying "This ticket is ready for engineering hand off." 

</p>
</details>

**Comment by melissahermes99** [comment]
@elizabeth-ux , @EchoProject , and I discussed this during the meeting tonight and we thought a good way to work around this issue could be muting the areas that are not a part of LA. For example, lightening the names of surrounding cities or removing the names altogether, and removing geographical features such as mountains and roads. I am not sure how feasible this solution is but I can work on some mockups of how that might look in the coming days.

**Comment by EchoProject** [comment]
@melissahermes99 Nich thinks we should be able to put an overlay over parts of the map. He's going to look into it further, and I think we can start mocking up designs for what this might look like. He mentioned we should consider cases like Beverly Hills and Culver City that are not a part of LA and look like they are within the city boundary.

**Comment by melissahermes99** [comment]
I created some concepts of what the map could look like. Since I'm not sure if we'll be able to remove geographical features, I figured these maps would be a good place to start as far as ideas go. 

![option 3](https://user-images.githubusercontent.com/97048386/185513979-afbb6ff9-af96-4773-8ee9-73d8ffa9406f.png)
![option 2](https://user-images.githubusercontent.com/97048386/185513983-e191d7bd-01b7-4e89-bc48-8df8d81863bb.png)
![option 1](https://user-images.githubusercontent.com/97048386/185513984-3f676f6f-5589-47a8-ad5e-77ae665b05d7.png)

**Comment by EchoProject** [comment]
I'm prefer the bottom one. We want to be sure that the map doesn't become too abstract such that a user wouldn't know it's the City of LA right away. The borders seem a little rough to me in black. Not sure if there's a solution. @melissahermes99 is there a way to soften the borders a little? Maybe make the darkened terrain not as dark?

**Comment by EchoProject** [comment]
@nichhk what are your thoughts on the above three options? This might be low-hanging fruit to get built.

**Comment by EchoProject** [comment]
@nichhk I think once we understand what the frontend capabilities are for this change, we can make a decision and move to development. Is there someone that can take over ownership of this issue?

**Comment by mc759** [comment]
@edwinjue Just wanted to resurface this issue as it was a conversation with Nich and Eric in October. We can discuss where it falls in the prioritization of dev tasks.

**Comment by mc759** [comment]
@edwinjue see list below for areas not included in LA City (I can point to each place on the last map that @melissahermes99 posted in August

Incorporated Cities:
- [ ] West Hollywood
- [ ] Beverly Hills
- [ ] Culver City
- [ ] Santa Monica
- [ ] Universal City
- [ ] San Fernando 
- [ ] Inglewood 

Unincorporated
- [ ] Marina Del Ray
- [ ] Unincorporated LA Southwest of Culver City (this includes View Park-Windsor Hills, Ladera Heights, and oil fields)
- [ ] Federal Land (VA Campus)
- [ ] Franklin Canyon (weird square above Beverly Hills)

Let me know if you have any questions or need visual assistants

**Comment by edwinjue** [comment]
Hi William, would you be interested in this issue as a follow-up to #1539? There are some areas on our map that are not officially a part of LA city and we may need to update the Map accordingly.

**Comment by wmb1232** [comment]
Yes I can take a look at this week and maybe work with you on ways to approach this

**Comment by wmb1232** [comment]
After looking at this for some time, I believe that we would need the geoJson coordinates for the places we are trying to differentiate, in the current code we do not have that information. This would be the first of a series of steps to create a layer that would display the cited places with some type of shadow that would signal they are not part of the available districts.

**Comment by ryanfchase** [comment]
Question for design. Is it possible that we could modify the boundary fill styles to clarify which regions are in Los Angeles?

<details><summary>Screenshot of Opaque NCs</summary>
<p>

![Image](https://github.com/user-attachments/assets/4d83ab11-c233-4fca-8fc8-6cac69070328)

</p>
</details>

**Comment by Joy-Truex** [comment]
Interesting! Yes i definitely think we can explore; relatively small design lift just have to ensure that the color contrast appropriately meets accessibility standards.
Please lmk if you'd like us to proceed with this.

**Comment by ryanfchase** [comment]
I've also found data that will highlight the boundaries for parks: https://data.lacity.org/Parks-Recreation/Department-of-Recreation-and-Parks-GIS-Map-of-Park/nuub-r4zx

**Comment by ryanfchase** [comment]
I think I will move forward with creating this ticket (will be ready later this week). Motivation: this will [address an insight brought up in the '24 Aug UXR reports](https://docs.google.com/presentation/d/1w1YpzC-sM7X1OOQ70f5Ey8h6Q0yOoTlC/edit#slide=id.p16). Although, I will mention that this wasn't addressed in our [Aug '24 Action Steps spreadsheet](https://docs.google.com/spreadsheets/d/14otyz2v-vYOnW9zJxpWTLSySTK4sJ2pDRO5wfre7e4Q/edit?gid=0#gid=0), so perhaps it didn't get prioritized.

I'm going to icebox this and wait until we can discuss in Product Planning. I have a couple of questions I need to ask dev lead and design lead.

**Comment by traycn** [comment]
Assigned to you @ryanfchase and added `Ready for dev lead` to add the issue to my proj board. I'll leave a follow up comment if I have any questions after reviewing the referenced docs. ETA 1/31 3PM.

**Comment by ryanfchase** [comment]
@traycn feel free to leave questions, but I am planning on addressing this at upcoming product planning.

**Comment by ryanfchase** [comment]
Adding `ready for product`, `draft`, I will want to modify this to take on the form of our design ticket. This ticket is a good candidate for the design ticket because previous designers have documented their work

Related Tickets
- https://github.com/hackforla/311-data/issues/1585
- https://github.com/hackforla/311-data/issues/1861

ETA: 2025-02-09

**Comment by ryanfchase** [comment]
Archiving the previous version of this ticket as of 2025-02-08

Title
```
Clarify regions that are not part of LA city on Map #1305
```

Body
```
### Overview
On the map, there are certain regions that are not part of the City of LA--e.g., West Hollywood, Beverly Hills, Culver City--but are entirely contained within the city. The only indication that they are not part of the city is that they have no request pins there and that hovering over them doesn't highlight them.

### Action Items
- [ ] Determine if there is a way to more clearly demonstrate that these regions are not part of the city. If there's not, the current design is fine.

### Resources
- #1396 
See dev.311-data.org to see an example of the current state.

### Dependency

- [x] Per @wmb1232: we would need the geoJson coordinates for the places we are trying to differentiate
- [x] I have a list of questions (WIP) for dev and design, adding to agenda for upcoming product planning meeting

```

**Comment by cottonchristopher** [comment]
Create ticket to research the listed city boundaries. (Dev ticket) https://github.com/hackforla/311-data/issues/1305#issuecomment-1459202092

**Comment by ryanfchase** [comment]
Passing this off to design lead. Please add the following labels:
- size
- complexity

If this ticket looks OK, please add
- ready for prioritization

**Comment by ryanfchase** [comment]
Prioritized. This ticket is ready to be picked up 👍

**Comment by pogwon** [comment]
A potential design solution we are exploring is for a popup to appear when a user hovers over an area that is outside the city of LA's boundaries. The popup would have the name of the city (i.e. Pasadena, Long Beach, etc) along with some text saying it is outside of Los Angeles. Is this something that could potentially be implemented?

**Comment by ryanfchase** [comment]
> A potential design solution we are exploring is for a popup to appear when a user hovers over an area that is outside the city of LA's boundaries. The popup would have the name of the city (i.e. Pasadena, Long Beach, etc) along with some text saying it is outside of Los Angeles. Is this something that could potentially be implemented?

@pogwon we cannot do this quite yet. Please move forward with the limitation that we cannot display the name or highlight boundaries of the non-NC boundaries to users, knowing that in a month or so we can have this information available. If you must have this data available in order to complete this ticket (#1305), then we will move this to Icebox and list #1948 as the blocker.
- New ticket: https://github.com/hackforla/311-data/issues/1948
- Moving this ticket back to in-progress, unless it should go to icebox

<details><summary>More Info</summary>
<p>

In the past, I have attempted to quickly gather the data required to display this information on a map. In some cases, like Culver City, it was really difficult to obtain the data in a format that our application can use. Publicly available boundary data is usually formatted for software like ArcGIS. I'm not familiar enough with these data formats to easily convert it to GeoJSON. I will need a developer with a data science or data analytics background get to work on this, it falls outside the skillset of web developers. So please take into consideration that we _can_ obtain the boundaries you're looking for, but it will take some time to actually have it working.

</p>
</details>

**Comment by pogwon** [comment]
### Three potential design solutions


1. Using a popup that would show up on hover over a non-NC area that would only say the current area is outside the boundaries

<img width="773" alt="Image" src="https://github.com/user-attachments/assets/0e89e6c3-ff06-4a25-8207-24fa45f602d8" />

---

2. A simpler text overlay that names the neighborhood name even while it is outside of the boundaries. However, unlike the darker blue change that occurs when hovering over an LA NC area, it would stay that lighter blue upon hovering.

<img width="392" alt="Image" src="https://github.com/user-attachments/assets/d83cc36c-f1b1-4bda-9045-45ad5e9d4d2f" />

---

3. Uses a popup design more consistent with #1807 but instead of the blue background, it uses gray to indicate a "disabled"/unusable state while showing the city name.

<img width="900" alt="Image" src="https://github.com/user-attachments/assets/73d68801-34e3-4b2d-9149-85a89c59b857" />

**Comment by pogwon** [comment]
After receiving feedback from both the design and dev teams, I went forward with the third design from V1 and revisited the initial design where non-NC areas were shaded out. I went with a lighter and more transparent gray that would appear upon hovering over the specific area along with a gray popup that would show the name of the area. After discussing with the design team, we concluded that this would be the most usable and easily understood design. To my understanding, this sort of design isn't able to be implemented right now but should be in the near future. Since this is for post-launch implementation, should it be tabled for now and placed in Icebox? @ryanfchase @Joy-Truex 

<img width="1173" alt="Image" src="https://github.com/user-attachments/assets/c46beed9-0914-4d87-aa62-d4ed04514a0b" />

**Comment by ryanfchase** [comment]
ETA for review 3/16

**Comment by ryanfchase** [comment]
@pogwon 

> To my understanding, this sort of design isn't able to be implemented right now but should be in the near future. Since this is for post-launch implementation, should it be tabled for now and placed in Icebox?

Let's proceed like so:
- you should complete your work on this ticket by completing any remaining hand off materials for the developers who will work on implementing this design. Let's proceed with the knowledge that your design is technically feasible, but we are awaiting verification that we can obtain any non-LA NCs that surround the NCs
- I will make the dev ticket to implement your design, but _that_ ticket will be put in the icebox. This will free you up, and prevent you from having to revisit this ticket at some arbitrary time in the future.

**Comment by ryanfchase** [comment]
I've moved this back to in-progress for the designer to complete work on the ticket.

**Comment by pogwon** [comment]
### V3 Changes:

- Added darker opaque border to non-NC boundaries to be more consistent with NC-area boundary designs
- Included example for 'Greater Los Angeles' area popup that for areas that are not detailed in [this list](https://github.com/hackforla/311-data/issues/1305#issuecomment-1459202092)

### Handoff Materials:

Link to Figma file: https://www.figma.com/design/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=17008-4662&t=uUUmjNzKiSvWFDbv-4

<img width="1437" alt="Image" src="https://github.com/user-attachments/assets/14ab5434-4c8a-4a26-9cfc-bb086d95e33a" />

**Comment by ryanfchase** [comment]
I'm seeing `ready for` labels, moving this to the In Review column

**Comment by Joy-Truex** [comment]
Hey @pogwon - from a design standpoint this is approved!


From a larger data standpoint, I would like to specify with a bit more clarity here + elsewhere what we are including in our map.

Is this our working definition of Greater Los Angeles?

Los Angeles-Long Beach, CA Combined Statistical Area (CSA)​
- Los Angeles County
- Orange County
- Ventura County
- Riverside County
- San Bernardino County

It is my opinion it is important for the design (ease of knowing what is included) and product (credibility and use case) implications to have approved and (official/gov defined) specified definitions of our regions listed in Wiki for:
- Neighborhood Councils
- Los Angeles City
-adjacent Los Angeles cities
- Greater Los Angeles

However this should perhaps be its own separate ticket to be taken on by research or product team.

Resources:
https://lacounty.gov/government/about-la-county/maps-and-geography/
https://www.whitehouse.gov/wp-content/uploads/2023/07/OMB-Bulletin-23-01.pdf

**Comment by ryanfchase** [comment]
@pogwon looks like we are awaiting an answer? I can add this to our general meeting to try and obtain concensus.

Also, worth noting, our data team has collected a large portion of the geojson boundaries needed to implement this design: 
- https://github.com/hackforla/311-data/issues/1948#issuecomment-2868239860

**Comment by pogwon** [comment]
> [@pogwon](https://github.com/pogwon) looks like we are awaiting an answer? I can add this to our general meeting to try and obtain concensus.
> 
> Also, worth noting, our data team has collected a large portion of the geojson boundaries needed to implement this design:
> 
>     * [DATA - Obtain geojson boundaries for areas outside City of Los Angeles #1948 (comment)](https://github.com/hackforla/311-data/issues/1948#issuecomment-2868239860)

Yes. I believe, the design itself has been approved, but as @Joy-Truex mentioned, we're just waiting to decide which non-NC areas we will label specifically vs relegate to general Greater Los Angeles area.

**Comment by ryanfchase** [comment]
I've gone ahead and created draft ticket to provide guidance on areas e.g. determine if it is a part of Greater Los Angeles, part of LA County, etc.
- https://github.com/hackforla/311-data/issues/1982

@Joy-Truex @pogwon should this ticket move to the icebox? Or is it that the dev hand off ticket should be iceboxed until have this official list? Either way makes sense to me, please let me know.

**Comment by Joy-Truex** [comment]
In my opinion the design is complete, but Dev implementation has a blocker of #1982

**Comment by ryanfchase** [comment]
FYI @Joy-Truex the relevant definitions have been added to the glossary ticket
- https://github.com/hackforla/311-data/issues/1982

I have moved the dev ticket out of the icebox and I will advancing it through the project board soon!
- https://github.com/hackforla/311-data/issues/1585
